src/option.h

```diff
@@ -228,6 +228,22 @@ namespace librealsense
         const char* get_description() const override { return option_description::get_description(); }
     };
 
+    class readonly_float_option : public float_option
+    {
+    public:
+        readonly_float_option(const option_range& range)
+            : float_option(range) {}
+
+        bool is_read_only() const override { return true; }
+
+        void set(float) override
+        {
+            throw not_implemented_exception("This option is read-only!");
```

Comment:
Log the option's name

Reply:
This is copied for the pre-existing readonly_option (see L48). Furthermore, the option object isn't aware of which enum its mapped from

Reply:
You can use the option's description if the enum is not available.
No need for fix in the L48